### PR TITLE
Removes "problemCells" from the code

### PR DESCRIPTION
**Pull request by DrPaulSharp**

### CODE REVIEW

File: API/parseClassToStructs.m [hunk 1]
```diff
@@ -370,6 +246,15 @@
 problemStruct.resolutionParams = inputStruct.resolutionParamValues; %inputStruct.resolutions;           % **** note resolutionParam workaround (todo) ****          
 problemStruct.params = inputStruct.paramValues;
 problemStruct.numberOfLayers = inputStruct.numberOfLayers;
+problemStruct.contrastLayers = inputStruct.contrastLayers;
+problemStruct.layersDetails = inputStruct.layerDetails;
+
+if isempty(inputStruct.files)
+    problemStruct.customFiles = {''};
```
[the remark this answers is not in the document]
Comment (StephenNneji) [reply API/parseClassToStructs.m:253]: It builds and runs when I set `problemStruct.customFiles = inputStruct.files;` without the empty check

**Review by StephenNneji — APPROVED** [review]
Look good, please see comments

File: examples/miscellaneous/bayesBenchmark/directCompare3D.m [hunk 2]
```diff
@@ -88,8 +88,8 @@
     probArray = probArray.probArray;
 else
 
-    [problemStruct,problemCells,problemLimits,priors,controls] = RatParseClassToStructs_new(d2oproblem,controls);
-    [problemStruct,fitNames] = packParams(problemStruct,problemCells,problemLimits,controls.checks);
+    [problemStruct,problemLimits,priors,controls] = RatParseClassToStructs_new(d2oproblem,controls);
```
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply examples/miscellaneous/bayesBenchmark/directCompare3D.m:91]: No, it was removed/renamed to `parseClassToStructs` a long time ago.

File: API/parseClassToStructs.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply API/parseClassToStructs.m:253]: Perfect, let's remove it then. Do the tests all pass?

[the remark this answers is not in the document]
Comment (StephenNneji) [reply API/parseClassToStructs.m:253]: Yes but you will need to rebuild the test data

**Pull request merged**